docs/sources/flow/reference/components/phlare.ebpf.md

```diff
@@ -0,0 +1,187 @@
+---
+title: phlare.ebpf
+labels:
+  stage: beta
+---
+
+# phlare.ebpf
+
+{{< docs/shared lookup="flow/stability/beta.md" source="agent" >}}
+
+`phlare.ebpf` configures an ebpf profiling job for the current host. The collected performance profiles are forwarded to
+the list of receivers passed in
+`forward_to`.
+
+Multiple `phlare.ebpf` components can be specified by giving them different labels, however it is not recommended.
+
+## Usage
+
+```river
+phlare.ebpf "LABEL" {
+  forward_to = RECEIVER_LIST
+}
+```
+
+## Arguments
+
+The component configures and starts a new ebpf profiling job to collect performance profiles from the current host.
+
+The following arguments can be used to configure a `phlare.ebpf`. Only the
+`forward_to` field is required and any omitted fields take their default
+values.
+
+| Name               | Type                     | Description                                                            | Default        | Required |
+|--------------------|--------------------------|------------------------------------------------------------------------|----------------|----------|
+| `forward_to`       | `list(ProfilesReceiver)` | List of receivers to send collected profiles to.                       |                | yes      |  
+| `targets`          | `list(map(string))`      | List of targets to group profiles by container id                      |                | no       |   
+| `default_target`   | `map(string)`            | Default target to use when a PID is not associated with a container id |                | no       |    
+| `targets_only`     | `bool`                   | A flag to ignore profiles not associated with a container id           | false          | no       |     
+| `job_name`         | `string`                 | The job name to override the job label with.                           | component name | no       |      
+| `collect_interval` | `duration`               | How frequently to collect profiles                                     | `10s`          | no       |       
+| `sample_rate`      | `int`                    | How many times per second to collect samples                           | 100            | no       |     
```

Comment:
This says the default sampling rate is 100Hz. This is a bit odd to me because it is usually recommended to use odd/prime sampling rates (e.g., 19Hz for parca, or 97Hz)  in order to reduce the likelihood that the sampling interrupt always/frequentlty overlaps with some other event occurring at a certain frequency. However, I was not able to quickly find any hard data on this. Was this considered / was 100Hz chosen for a particular reason?

Reply:
made it 97, thanks